frontend/e-buddy/src/context/AuthContext.tsx

```
@@ -20,37 +47,109 @@ const AuthContext = createContext<AuthContextType | undefined>(undefined);
 
-  useEffect(() => {
-    // Check token validity on app load
-    const token = localStorage.getItem("auth_token");
-    if (token) {
+  // Clear auth state
+  const logout = useCallback(() => {
+    localStorage.removeItem("auth_token");
+    localStorage.removeItem("refresh_token");
+    setIsAuthenticated(false);
+    setUser(null);
+    delete api.defaults.headers.common["Authorization"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)